maple-sdk/usage-guide.md

```diff
@@ -0,0 +1,53 @@
+# Usage Guide
+
+This guide will help you understand how to use the Maple SDK for various tasks, such as connecting to contracts and executing transactions.
+
+# Overview
+
+The Maple SDK simplifies interacting with Maple Finance's smart contracts on Ethereum blockchain. It includes address mappings for each contract to facilitate deploying to different networks: `mainnet-prod`, `mainnet-dev`, `goerli-prod` and `goerli-dev`.
```

Comment:
We should explain and differentiate between networks (mainnet/goerli) and our environments (dev/prod)